src/query/__tests__/buildRQL-test.js

```diff
@@ -190,6 +192,20 @@ describe('buildRQL', () => {
       expect(node1 === node2).toBe(false);
     });
 
+    it('returns different queries for the same component if cache is disabled', () => {
+      DisableRelayQueryCache.disableCache();
+      const builder = Component => Relay.QL`
+        query {
+          node(id:$id) {
+            ${Component.getFragment('foo')}
+          }
+        }
+      `;
+      const node1 = buildRQL.Query(builder, MockContainer, 'foo', {id: null});
+      const node2 = buildRQL.Query(builder, MockContainer, 'foo', {id: null});
+      expect(node1 === node2).toBe(false);
```

Comment:
could do `expect(node1).not.toBe(node2)`, which would provide a slightly more useful error message (the value1 !== value2 instead of true !== false)